### PR TITLE
Fix inputs/outputs of webapp tasks

### PR DESCRIPTION
**Pull request by timroes**
## What

Fix the input/outputs of all webapp gradle tasks to make them build more incremental.

## Can this PR be safely reverted / rolled back?
*If you know that your PR is backwards-compatible and can be simply reverted or rolled back, check the YES box.*

*Otherwise if your PR has a breaking change, like a database migration for example, check the NO box.*

*If unsure, leave it blank.*
- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-webapp/build.gradle [hunk 1]
```diff
@@ -46,8 +52,15 @@ task pnpmBuild(type: PnpmTask) {
 
     inputs.files commonConfigs
     inputs.file '.eslintrc.js'
+    inputs.file '.stylelintrc'
+    inputs.file 'scripts/calculate-source-hash.sh'
+    inputs.file 'scripts/load-declarative-schema.sh'
```
Comment (josephkmh) [inline airbyte-webapp/build.gradle:57]: Maybe worth adding the whole `scripts` folder? There are some dev scripts in there that don't matter, but I can imagine we will add some other build-relevant scripts there later and forget to add them here.

File: airbyte-webapp/build.gradle [hunk 2]
```diff
@@ -46,8 +52,15 @@ task pnpmBuild(type: PnpmTask) {
...
+    inputs.file 'orval.config.ts'
+    inputs.file 'vite.config.ts'
+    inputs.file 'index.html'
```
Comment (josephkmh) [inline airbyte-webapp/build.gradle:60]: Should we add `.env` to this list?

**Review by josephkmh — APPROVED** [review]
Looks good, just a couple suggestions of additional input files for the build

**Comment by timroes** [comment]
Superseeded by https://github.com/airbytehq/airbyte-platform-internal/pull/4811

File: airbyte-webapp/build.gradle (hunk 2 above)
Comment (timroes) [reply airbyte-webapp/build.gradle:60]: `.env` is already part of that list via `commonConfigs` above.

**Pull request closed without merging**